### PR TITLE
Added build script for ajstarks/svgo

### PR DESCRIPTION
**Pull request by SunidhiGaonkar**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by seth-priya** [comment]
@prashantkhoje please review

File: a/ajstarks_svgo/ajstarks_svgo_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	
+fi
+
+
+if ! go test ./... -v ; then
+	echo "------------------Test_fails---------------------"
+	exit 1
+else
+	echo "------------------Test_success-------------------------"
+
```
Comment (prashantkhoje) [inline a/ajstarks_svgo/ajstarks_svgo_ubi_8.5.sh:62]: Please remove the blank line.

Comment (SunidhiGaonkar) [reply a/ajstarks_svgo/ajstarks_svgo_ubi_8.5.sh:60]: I have made the required changes, please review. @prashantkhoje

Comment (prashantkhoje) [reply a/ajstarks_svgo/ajstarks_svgo_ubi_8.5.sh:60]: LGTM.
The Travis CI build hasn't finished yet though.

**Pull request merged**